apps/site/lib/site_web/templates/layout/_footer.html.eex

```diff
@@ -1,107 +1,70 @@
-          <li class="m-footer__item"><%= link "Developers", to: cms_static_page_path(@conn, "/developers"), data: [scroll: true] %></li>
-          <li class="m-footer__item"><%= link "Financials", to: cms_static_page_path(@conn, "/financials"), data: [scroll: true]  %></li>
-          <li class="m-footer__item"><%= link "Innovation", to: cms_static_page_path(@conn, "/innovation") %></li>
-          <li class="m-footer__item"><%= link "Engineering Design Standards", to: cms_static_page_path(@conn, "/engineering/design-standards-and-guidelines") %></li>
-        </ul>
-      </li>
-      <li class="m-footer__group m-footer--mobile__group">
-        <div class="m-footer__group-title">Customer Support</div>
-        <ul class="list-unstyled m-footer--mobile__items">
-          <li class="m-footer__item m-footer--mobile__item">
-            <%= link "Contact Us", to: customer_support_path(@conn, :index), data: [scroll: true]  %>
-          </li>
-          <li class="m-footer__item m-footer--mobile__item">
-            <%= link "Lost and Found", to: cms_static_page_path(@conn, "/customer-support/lost-and-found"), data: [scroll: true]  %>
-          </li>
-          <li class="m-footer__item m-footer--mobile__item">
-            <%= link "Policies", to: cms_static_page_path(@conn, "/policies"), data: [scroll: true]  %>
-          </li>
-          <li class="m-footer__item hidden-sm-down"><%= link "Request Public Records", to: "https://massachusettsdot.mycusthelp.com/WEBAPP/_rs/supporthome.aspx?lp=3&COID=64D93B66", target: "_blank" %></li>
-          <li class="m-footer__item hidden-sm-down"><%= link "Civil Rights", to: cms_static_page_path(@conn, "/policies/title-vi"), data: [scroll: true]  %></li>
-        </ul>
-      </li>
-      <li class="m-footer__group">
-        <div class="m-footer__group-title">Follow Us Online</div>
-        <ul class="c-social">
-            <% social_links = [
-                {"facebook", "https://www.facebook.com/TheMBTA/", []},
-                {"instagram", "https://www.instagram.com/thembta/", []},
-                {"linkedin", "https://linkedin.com/company/mbta", []},
-                {"youtube", "https://www.youtube.com/user/mbtagm/", []},
-                {"twitter", "https://twitter.com/MBTA", []},
-                {"tiktok", "https://www.tiktok.com/@thembta", [font_family: "fab"]}
-             ] %>
-          <%= for {name, href, attributes} <- social_links do %>
-            <li class="c-social__item m-footer__item">
-              <%= link [
-                fa(name, attributes),
-                content_tag(:span, "MBTA " <> String.capitalize(name), class: "sr-only")
-              ], to: href, class: "c-social__link c-social--" <> name, target: "_blank" %>
-            </li>
-          <% end %>
-        </ul>
+<div class="all-container">
+  <address class="call-us-container">
+    <div class="container m-footer__links">
```

Comment:
**question:** The `.m-footer__links` class was originally made to handle the layout of the three columns of links in the other section (switching it from grid-based 3 columns to 1 column depending on breakpoint)... is there a benefit to reusing it here?

Reply:
**similar question:** The same goes for the `.container` class which is at various places in this footer -- its main purpose as it's used elsewhere in the site is to contain the content to a certain page width; I don't think it make sense to use it for various pieces of the footer that aren't usually supposed to be wide across the whole page anyway.

Reply:
I used the `.m-footer__links` so that on medium breakpoints the rows for the contact numbers and links would match exactly. Can work on removing `.container`